v2/README.md

```diff
@@ -11,6 +11,11 @@ Instead of deploying and managing your Azure resources separately from your Kube
 ## Project Status
 This project is stable. We follow the [Kubernetes definition of stable](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#feature-stages).
 
+| Release                                                                       | Date       |                 |
```

Comment:
Is this intended to be a list of... recent releases? Releases in the last year? All releases?

Might help to have a header clarifying what we are showing here?

Reply:
As mentioned in my other comment: This information is already available in a standardized place on GitHub, the [releases tab](https://github.com/Azure/azure-service-operator/releases). Since it's part of the GitHub standard UI, folks know to navigate there to check out releases. There's also already a shortcut to it on the main page in a standardized place.

What do you think about just linking to that location here, rather than making our own table that just mirrors what's already automatically included in the releases UI?

Reply:
I took a look around at a few other projects and I don't see a list of releases in the readme of CAPZ, Azure Workload Identity, cert-manager, or Kubernetes.

cert-manager does have a list [here](https://cert-manager.io/docs/installation/supported-releases/), although the context is a bit different because there it's saying which versions are supported whereas in our case I think our only officially supported version is whatever the latest release is.

I suspect Kubernetes also has a manually maintained list somewhere although in a few minutes of searching I can't find it.

Reply:
Part of the issue I'm trying to address is caused by our reuse this README as the landing page of our website as well, where there is not a convenient list of existing versions/releases.

Maybe a link would be good enough? I'm unsure.

I very nearly titled the first column **Supported Releases**, but wasn't sure about doing that.

Reply:
I guess my point is, none of the repos I checked have a list of versions on the front page of their site either. Obviously that doesn't prevent us from putting a list there, but is it what people are expecting? I would feel differently if we actually did have a list of supported versions, but I think by most definitions of supported we currently only support the latest release.

I just don't want us to have to manually maintain something that we then end up forgetting to update, as I think having a list and having it be out of date is a worse look than just not having a list at all, and there are already a number of "standard" ways to get versions such as:
1. The GitHub releases page
2. Helm commands to list releases (if you're using Helm).